src/contracts.js

```diff
@@ -118,6 +124,23 @@ export const ContractList = () => (
         label="Status"
         render={(record) => (record.active === true ? "Active" : "Inactive")}
       />
+      {HasPermissions("contracts", "read") && (
+        <ArrayField source="paymentSettlement" label="Salary">
+          <SingleFieldList>
+            <CustomizableChipField source="salary">
+              {(record) => {
+                if (record) {
+                  const label = `${record.currency}${
+                    record.salary
+                  }/${record.modality.charAt(0).toLowerCase()}`;
+                  return label;
```

Comment:
For Developer and Analyst role (they cannot see salary), i saw the value of ${record.salary} is null, so i think we could improve this with something like:
```
if (record) {
  const salary = `${record.salary}` === "null" ? '-' : `${record.salary}`
  return `${record.currency} ${salary} /${record.modality.charAt(0).toLowerCase()}`;
}
```